src/private/MDMTransaction+Private.h

```diff
@@ -19,8 +19,18 @@
 @protocol MDMPlan;
 
 @interface MDMTransactionLog : NSObject
-@property(nonatomic) NSArray<id<MDMPlan>> *plans;
-@property(nonatomic) id target;
+
+@property(nonatomic, copy) NSArray<id<MDMPlan>> *plans;
+@property(nonatomic, strong) id target;
+@property(nonatomic, copy) NSString *name;
```

Comment:
I'm not sure if it makes sense anymore to have a `plans` array *and* a `name` field. Since multiple plans cannot share the same name (right @jverkoey ?) Not sure if this is in scope for this PR